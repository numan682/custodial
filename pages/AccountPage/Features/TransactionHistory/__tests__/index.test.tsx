import * as React from 'react'
import { render } from '@testing-library/react'

import { TransactionHistory } from '..'

describe('<TransactionHistory  />', () => {
  it.skip('should match snapshot', () => {
    const component = render(<TransactionHistory />)
    expect(component.container.firstChild).toMatchSnapshot()
  })
})
