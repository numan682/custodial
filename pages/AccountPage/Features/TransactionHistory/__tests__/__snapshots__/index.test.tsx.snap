// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TransactionHistory  /> should match snapshot 1`] = `<div />`;
